### ISSUE TITLE
New Twitter UI shows up for mobile.twitter.com

### ISSUE THREAD

**Issue by arturo182** [Bl4Cc4t/GoodTwitter2#104]
I got around this by installing this script that redirects automatically from mobile.*.com websites: https://greasyfork.org/en/scripts/33171-this-ain-t-a-phone

**Comment by Bl4Cc4t** [comment]
I’m not really sure how this is related to this project, you can use the script you mentioned for that purpose :)

**Comment by arturo182** [comment]
I thought it was related in the sense that GoodTwitter 2 doesn't work with mobile version of twitter when accessing it on desktop (which can happen when a friend sends you a link from their mobile app). So I thought it would be helpful to post the other script (not mine) if someone comes here to complain about this issue :)

**Comment by geyerbri** [comment]
This has been sitting here for a couple years (and is really more of an enhancement than a bug), so I went ahead and wrote a small snippet of code to deal with this. I do agree that it would be a good automatic feature, so incorporating it into this script, rather than requiring yet another one to be installed, seems like something worthwhile.

But maybe you wouldn't want it automatically implemented, and instead it should be a new setting to toggle on or off? I don't know enough about the script to write in a new feature toggle for the settings, so unfortunately the pull request I just submitted doesn't have a settings menu entry. Hopefully, if you think it should be toggled and not automatic, adding such a toggle would be trivial.

**Comment by Bl4Cc4t** [comment]
Good idea! I don't know why I never considered adding a toggle for this, I even find myself changing the url sometimes...
